lib/mozart_fetcher/http_client.ex

```diff
@@ -36,8 +36,10 @@ defmodule HTTPClient do
     end
   end
 
-  defp set_request_headers("https://fabl.api." <> _),
-    do: [{"accept-encoding", "gzip"}, {"ctx-unwrapped", "1"}]
+  defp set_request_headers("https://fabl.api." <> _) do
+    service_env = if MozartFetcher.environment() === :prod , do: "live", else: "test"
```

Comment:
The environment here is "prod" for both test and live environments. I don't think we set an environment variable for test/live in Fetcher but we seem to in other Mozart applications.